lib/evmone/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 include(LibraryTools)
 
 hunter_add_package(intx)
-find_package(intx CONFIG REQUIRED)
+find_package(intx 0.4.0 CONFIG REQUIRED)
```

Comment:
What is the purpose of this? Dependencies versions are pinned by Hunter anyway.

Reply:
I wanted to ensure at least 0.4.0 is used, which is required for `as_bytes`, but if it is accomplished by hunter, then this is not needed.

Reply:
Remove it please. I will handle that separately.

Reply:
Done.